wp-content/plugins/hurumap/api.php

```diff
@@ -84,6 +84,18 @@ function register_routes()
     ));
 }
 
+function get_domain()
+{
+  $url = site_url();
+  $pieces = parse_url($url);
+  $domain = isset($pieces['host']) ? $pieces['host'] : $pieces['path'];
+  if (preg_match('/(?P<domain>[a-z0-9][a-z0-9\-]{1,63}\.[a-z\.]{2,6})$/i', $domain, $regs)) {
```

Comment:
:scream: Lets just use `takwimu.africa` for now but lets have a story to add an option where someone can enter the main domain name to set to. It's hard to know what is the main/subdomain e.g. takwimu.africa, takwimu.co.tz, etc.